openedx/core/lib/hash_utils.py

```diff
@@ -0,0 +1,23 @@
+"""
+Utilities related to hashing
```

Comment:
If we're pulling this from the DOP library, I would include more information in this docstring about why we copied it over and what situations it's used for. I don't want anyone to get mistaken and believe it's more secure than it is.

Reply:
okay, good point... though I might look into whether we can make it securer